swtbot_test/src/kerneltestsuites/kernelCCRXKernelGUI.java

```diff
@@ -0,0 +1,378 @@
+			}
+		}
+		
+		for (SWTBotTreeItem config : kernelConfigTree) {
+			if (config.cell(0).contains("Maximum syscall interrupt priority")) {
+				bot.sleep(2000);
+				config.click(1);
+				config.click(1);
+				Utility.clickClearConsole();
+				bot.text(1).setText("3");
+				isConsoleContain2 = !Utility.isConsoleHasString(
+						"E04020001: Value must be in range configKERNEL_INTERRUPT_PRIORITY + 1 ~ configMAX_PRIORITIES - 1");
+			}
+		}
+		
+		for (SWTBotTreeItem config : kernelConfigTree) {
+			if (config.cell(0).contains("Maximum syscall interrupt priority")) {
+				bot.sleep(2000);
+				config.click(1);
+				config.click(1);
+				bot.text(1).setText("4");
+				isConsoleContain3 = !Utility.isConsoleHasString(
+						"E04020001: Value must be in range configKERNEL_INTERRUPT_PRIORITY + 1 ~ configMAX_PRIORITIES - 1");
+			}
+		}
+		
+		for (SWTBotTreeItem config : kernelConfigTree) {
+			if (config.cell(0).contains("Maximum syscall interrupt priority")) {
+				bot.sleep(2000);
+				config.click(1);
+				config.click(1);
+				bot.text(1).setText("6");
+				isConsoleContain4 = !Utility.isConsoleHasString(
+						"E04020001: Value must be in range configKERNEL_INTERRUPT_PRIORITY + 1 ~ configMAX_PRIORITIES - 1");
+			}
+		}
+		
+		for (SWTBotTreeItem config : kernelConfigTree) {
+			if (config.cell(0).contains("Maximum syscall interrupt priority")) {
+				bot.sleep(2000);
+				config.click(1);
+				config.click(1);
+				bot.text(1).setText("7");
+				isConsoleContain5 = !Utility.isConsoleHasString(
+						"E04020001: Value must be in range configKERNEL_INTERRUPT_PRIORITY + 1 ~ configMAX_PRIORITIES - 1");
+			}
+		}
+		
+		for (SWTBotTreeItem config : kernelConfigTree) {
+			if (config.cell(0).contains("Maximum syscall interrupt priority")) {
+				bot.sleep(2000);
+				config.click(1);
+				config.click(1);
+				bot.text(1).setText("8");
+				isConsoleContain6 = Utility.isConsoleHasString(
+						"E04020001: Value must be in range configKERNEL_INTERRUPT_PRIORITY + 1 ~ configMAX_PRIORITIES - 1");
+			}
+		}
+		
+		if (!isConsoleContain1 || !isConsoleContain2 || !isConsoleContain3 || !isConsoleContain4 || !isConsoleContain5
+				|| !isConsoleContain6) {
+			assertFalse(true);
+		}
+	}
+	
+	@Test
+	public void tc_04_CheckDisableProperty() throws Exception {
+		SWTBotTreeItem[] kernelConfigTree = bot.tree(2).getTreeItem("Configurations ").getItems();
+		boolean isNotEditable1 = false;
+		boolean isNotEditable2 = false;
+		boolean isNotEditable3 = false;
+		boolean isNotEditable4 = false;
+		boolean isNotEditable5 = false;
+		boolean isNotEditable6 = false;
+
+		for (SWTBotTreeItem config : kernelConfigTree) {
+			if (config.cell(0).contains("The frequency of the CPU clock")) {
+				bot.sleep(2000);
+				config.click(1);
+				config.click(1);
+				try {
+					bot.text(config.cell(1)).setText("");
+				} catch (Exception e) {
+					isNotEditable1 = true;
```

Comment:
Q: is there any other way to check disable state, instead of using setText

Reply:
I don't think there is any better way to check disable, since disabled the difference with enabled is : 1: The color is lighter black than enable config and 2: It cannot be set a different value than default
And since we can both click on disable config and enable config, the method I implement is using the difference number 2 above, or else I think we have to compare the color of the text